Docs/how_to_build_on_linux.md

```diff
@@ -1,136 +1,194 @@
+	* __Build UE4.22:__ Needed to build the project. 
+* [CARLA build](#carla-build): 
+	* __Clone repository:__ Get the CARLA project in a local computer. 
+	* __Get assets:__ Download the assets, which are stored separately. 
+	* __Set the environment variable:__ Ensures communication between CARLA and Unreal Engine.
+	* __make CARLA:__ Build the server, client and packages. 
+
+The build process can be quite long and tedious. This documentation tries to make things clear, but the CARLA forum is open to anybody who wants to post unexpected issues, doubts or suggestions. There is a specific section for installation issues in Linux. Feel free to login and become part of the community. 
+
+<div class="build-buttons">
+<!-- Latest release button -->
+<p>
+<a href="https://forum.carla.org/" target="_blank" class="btn btn-neutral" title="Go to the latest CARLA release">
+CARLA forum</a>
+</p>
+</div>
+
+---------------
+##Requirements
+
+<h4>System specifics</h4>
+* __Ubuntu 16.04 or later:__ migration to Ubuntu 18 is slowly coming forward, but today's date there is still support for 16.04. 
+* __Two TCP ports:__ 2000 and 1 by default. Be sure neither firewall nor any other application are blocking these. 
+* __An adequate GPU:__ CARLA aims for realistic simulations, so the server needs at least a 4GB GPU. A dedicated GPU is highly recommended for machine learning. 
+
+
+<h4>Dependencies</h4>
+
+CARLA needs many dependencies to run. Some of them are built automatically during this process, such as _Boost.Python_. Others are binaries that should to be installed before starting the build (_cmake_, _clang_, different versions of _Python_ and much more). In order to do so, run the commands below in a terminal window. 
 
 ```sh
+sudo apt-get update &&
+sudo apt-get install wget software-properties-common &&
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test &&
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add - &&
+sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main" &&
 sudo apt-get update
-sudo apt-get install wget software-properties-common
-sudo add-apt-repository ppa:ubuntu-toolchain-r/test
-wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
-sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main"
-sudo apt-get update
-sudo apt-get install build-essential clang-7 lld-7 g++-7 cmake ninja-build libvulkan1 python python-pip python-dev python3-dev python3-pip libpng16-dev libtiff5-dev libjpeg-dev tzdata sed curl unzip autoconf libtool rsync
-pip2 install --user setuptools
-pip3 install --user setuptools
 ```
 
-!!! tip
-    For **Ubuntu 18.04**, change `libpng16-dev` to `libpng-dev` from the previous example.
+!!! important
+    The following commands differ depending on Ubuntu version. The only change is `libpng16-dev` which becomes `libpng-dev`, but the full set of commands is here twice to ease copying them. 
```

Comment:
While the only change is `libpng16-dev` (which becomes `libpng-dev`), the full set of commands is here for easier copying.